### ISSUE TITLE
Toast components appearing empty without message.

### ISSUE THREAD

**Issue by bnapky** [react-materialize/react-materialize#623]
### Description
The Toast component is showing up without any message inside.

### Steps to Reproduce
1. Set up React Materialize 
2. Import Toast component

Code Sample: 

```
import React from 'react';
import {Toast} from 'react-materialize'

const HomePage = () =>(<Toast toast="hello">Hello</Toast>);
export default HomePage;
```

**Expected behavior:** On click, a toast should appear with a message, in this case "hello"

**Actual behavior:** A toast appears empty, no message within it.

### Versions

**react-materialize:** 2.3.3
**materialize-css:** 1.0.0-rc.2
**react:** 3.6.0

I will be sending a PR addressing this issue.
As it seems, Materialize have changed their method to invoke a Toast.

- It changed from: `M.toast('Hello')`

- To: `M.toast({html: 'Hello'})`

**Comment by bnapky** [comment]
I've opened a [Pull Request](https://github.com/react-materialize/react-materialize/pull/624) for this.

**Comment by petergarnaes** [comment]
Hey, as you mentioned, the materialize API has had many breaking changes from `0.100.2` to `1.0.0-rc-2`. However the current version of `react-materialize` (v2.1.2) is meant to be used with materialize `v0.100.2`. As stated in the README, development of `v3.0.0` (for materialize `v.1.0.0-rc.2`) is happening on the `3.0.0-beta.0` branch.

Your issue is already fixed in #610, so please close this issue. Feel free to contribute to the 3.0.0 branch, just pick any of the open issues relating to 3.0.0 :wink:

**Comment by bnapky** [comment]
I completely missed that, thanks @petergarnaes for clarifying. I will be closing this issue then.
